pyg_lib/csrc/sampler/cpu/mapper.h

```diff
@@ -10,6 +10,8 @@ namespace sampler {
 template <typename scalar_t>
 class Mapper {
  public:
+  using type = scalar_t;
```

Comment:
why need this using?

Reply:
It's previously to deduce the element type, but now it's not used and I will remove it